### PR TITLE
Add scaladocs #patch

### PR DESCRIPTION
**Pull request by andresgomezfrr**
# TL;DR
Add scala docs to scala classes.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin
 - [x] Docs

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: flytekit-scala_2.13/src/main/scala/org/flyte/flytekitscala/SdkBindingData.scala [hunk 1]
```diff
@@ -28,6 +28,7 @@ import org.flyte.flytekit.{SdkBindingData => SdkJavaBindinigData}
 import java.time.{Duration, Instant}
 import scala.collection.JavaConverters._
 
+/** */
```
Comment (narape) [inline flytekit-scala_2.13/src/main/scala/org/flyte/flytekitscala/SdkBindingData.scala:31]: Is this the scaladoc equivalent to scala's `???` ;) 
I just commented the java equivalents methods, we can wait here to copy the docs, so we reuse work and increase consistency

Comment (andresgomezfrr) [reply flytekit-scala_2.13/src/main/scala/org/flyte/flytekitscala/SdkBindingData.scala:31]: yeah make sense!

Comment (andresgomezfrr) [reply flytekit-scala_2.13/src/main/scala/org/flyte/flytekitscala/SdkBindingData.scala:31]: > Is this the scaladoc equivalent to scala's ??? ;)

I forgot this class 🙈

**Review by narape — APPROVED** [review]
